### PR TITLE
Remove the JIT backend.

### PR DESCRIPTION
**Pull request by ryzhyk**
Remove the JIT backend and all the code that depends on it throughout the project.

Is this a user-visible change (yes/no):  no

### CODE REVIEW

**Review by mihaibudiu — APPROVED** [review]

File: Earthfile [hunk 1]
```diff
@@ -515,29 +481,20 @@ build-pipeline-manager-container:
     # First, copy over the artifacts built from previous stages
     RUN mkdir -p database-stream-processor/sql-to-dbsp-compiler/SQL-compiler/target
     COPY +build-manager/pipeline-manager .
```
[the remark this answers is not in the document]
Comment (ryzhyk) [reply Earthfile:483]: I think this is correct. It's the pipeline manager executable.  Can't build a pipeline manager container without it.

File: crates/pipeline_manager/src/bin/local-runner.rs [hunk 2]
```diff
@@ -42,7 +36,7 @@ async fn main() {
     .unwrap();
     let db = Arc::new(Mutex::new(db));
     let _local_runner = spawn(async move {
-        local_runner::run(db, &local_runner_config, &compiler_config).await;
+        local_runner::run(db, &local_runner_config.clone()).await;
```
Comment (lalithsuresh) [inline crates/pipeline_manager/src/bin/local-runner.rs:39]: Is .clone() required?

File: crates/pipeline_manager/src/bin/pipeline-manager.rs [hunk 3]
```diff
@@ -77,15 +77,14 @@ async fn main() -> anyhow::Result<()> {
...
     let _compiler = tokio::spawn(async move {
-        Compiler::run(&compiler_config_clone, db_clone)
+        Compiler::run(&compiler_config.clone(), db_clone)
             .await
             .unwrap();
     });
     let db_clone = db.clone();
     let _local_runner = tokio::spawn(async move {
-        local_runner::run(db_clone, &local_runner_config, &compiler_config).await;
+        local_runner::run(db_clone, &local_runner_config.clone()).await;
```
Comment (lalithsuresh) [inline crates/pipeline_manager/src/bin/pipeline-manager.rs:87]: Same question, are these clones required?

**Review by lalithsuresh — APPROVED** [review]

File: crates/pipeline_manager/src/bin/pipeline-manager.rs (hunk 3 above)
Comment (ryzhyk) [reply crates/pipeline_manager/src/bin/pipeline-manager.rs]: Must have inherited this with reverting some of the JIT commits.

**Pull request merged**